README.md

```diff
@@ -1,20 +1,27 @@
-# terraform-provider-logdna 
+# Terraform Provider for LogDNA
 
-🚧 Work in progress 🚧
+[![CircleCI](https://circleci.com/gh/logdna/terraform-provider-logdna/tree/master.svg?style=svg)](https://app.circleci.com/pipelines/github/logdna/terraform-provider-logdna)
+[![Coverage Status](https://coveralls.io/repos/github/logdna/terraform-provider-logdna/badge.svg?branch=main)](https://coveralls.io/github/logdna/terraform-provider-logdna?branch=main)
+
+🚧 In public beta 🚧
+
+[LogDNA](https://logdna.com) is a centralized log management platform. The LogDNA Provider allows organizations to manage Views and Alerts programmatically via Terraform.
+
+The [official docs for the LogDNA terraform provider](https://registry.terraform.io/providers/logdna/logdna/latest/docs) can be found in the Terraform registry.
 
 ## Example Terraform Configuration
 ```
```

Comment:
I feel like we should add a sentence here saying something like: 
"This example code for creating a new View with an Alert is in [HCL (Hashicorp Configuration Language)](https://www.terraform.io/docs/configuration/syntax.html)."

Reply:
@tanberry i feel that we don't need to explicitly mention that the code is written in hcl given that in the terraform registry- hashicorp provides the user with a using providers and learn terraform section:

<img width="1424" alt="Screen Shot 2020-10-27 at 4 28 20 PM" src="https://user-images.githubusercontent.com/10788080/97372964-0db16500-1872-11eb-8efc-65b304cd595e.png">